src/coreclr/vm/appdomain.cpp

```diff
@@ -1139,7 +1139,7 @@ void SystemDomain::Init()
 
     // At this point m_SystemDirectory should already be canonicalized
     m_BaseLibrary.Append(m_SystemDirectory);
-    if (!m_BaseLibrary.EndsWith(DIRECTORY_SEPARATOR_CHAR_W))
+    if (!m_BaseLibrary.EndsWith(SString{ DIRECTORY_SEPARATOR_CHAR_W }))
```

Comment:
I agree it should be simple, but I'm trying to avoid doing a lot of addtional work here given my plan is to start iterating on specific groups of methods. I will address these first so am going to defer them for now.